services/tinlake/actions.ts

```diff
@@ -95,6 +95,19 @@ export const mintNFT = async (
   }
 }
 
+export const updateNftFeed = async (
+  tinlake: ITinlake,
+  nftFeedId: string,
+  value: string,
+  riskGroup: string
+): Promise<PendingTransaction> => {
+  try {
```

Comment:
You should probably just `return await tinlake.updateNftFeed(nftFeedId, value, riskGroup)` here, rather than the whole try catch. The handler in the transactions duck automatically takes care of catching exceptions. The only places we should use the try-catch with `loggedError` in actions (and maybe I haven't consistently applied this yet), is for tinlake calls that are not the final call in the action.

Reply:
Makes sense, removed in https://github.com/centrifuge/tinlake-ui/pull/500/commits/d75187e44dc93cb14819d156198f746afb6681d3 . Can you please review that?